resources/user-agents/browsers/android-webview/android-webview-4-0.json

```diff
@@ -72,7 +72,8 @@
             "Pixel C": "Google Pixel C",
             "SM-G935F": "Samsung SM-G935F",
             "SM-G950F": "Samsung SM-G950F",
-            "SM-T820": "Samsung SM-T820"
+            "SM-T820": "Samsung SM-T820",
+            "PRA-LX1": "Huawei PRA-LX1"
```

Comment:
Please sort the list (by the token), most editors allow you to sort lines, which should do the trick.